File: Assignments/Assignment_13/p10.c
#include<stdio.h>
void main(){

        int row,count=0;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=row;i>0;i--){

                for(int j=1;j<=i;j++){

                        if(row%2==0){

                                if((i+j)%2!=0){

                                        printf("%d\t",(64+j+count));
                                }else{

                                        printf("%c\t",(64+j+count));
                                }
                        }else{

                                if((i+j)%2==0){

                                        printf("%c\t",(64+j+count));
                                }else{

                                        printf("%d\t",(64+j+count));
                                }
                        }
                }
                printf("\n\n");
                count++;
        }
}

File: Flutter/Assignment/Square pattern set-2/p10.dart
import "dart:io";
void main(){
	print("Enter the no. of rows:");
	int? row=int.parse(stdin.readLineSync()!);
	int x=1;
	for(int i=0;i<row;i++){
		int z=row-i;
		int y=z;
		for(int j=0;j<=i;j++){
			if(j==0){
				stdout.write("${x}\t");
			}else{
				stdout.write("${x-z}\t");
				z=z+(++y);
			}	
		}
		x=x+row-i;
		print("");
	}
}
